tswow-core/Public/TSGlobal.h

```diff
@@ -20,6 +20,8 @@
 #include "TSItem.h"
 #include "TSBase.h"
 #include "TSString.h"
+TSItemTemplate TC_GAME_API CreateItemTemplate(uint32 entry,uint32 copyItemID = 38);
```

Comment:
38 is a plain shirt incase people are unsure of what they want as a parent item and since there are so many minor fields it seemed easier to give it a parent even if a scripter doesnt want to. The real way it should be done is having the scripter create their own blank items in datascripts and using that for most usecases.

Reply:
addressed in the global.d.ts comments